commands/src/main/java/org/wildfly/extras/creaper/commands/orb/AuthValues.java

```diff
@@ -0,0 +1,22 @@
+package org.wildfly.extras.creaper.commands.orb;
+
+public enum AuthValues {
+    NONE("None"),
+    SERVER_AUTH("ServerAuth"),
+    CLIENT_AUTH("ClientAuth"),
+    MUTUAL_AUTH("MutualAuth");
+
+    private final String value;
+
+    AuthValues(String auth) {
+        this.value = auth;
+    }
+
+    public String toString() {
+        return value;
+    }
+
+    public String getValue() {
+        return value;
+    }
```

Comment:
Why do you need this `public`? Either this method should be package-private or the `value` field should be package-private. I guess this also applies to other enums in this PR.